### PR TITLE
Related to #99. Clarifies that there is no leak.

### PR DESCRIPTION
**Pull request by agutenkunst**

### CODE REVIEW

File: pilz_trajectory_generation/src/path_circle_generator.cpp [hunk 1]
```diff
@@ -41,15 +41,14 @@ std::unique_ptr<KDL::Path> PathCircleGenerator::circleFromCenter(
   KDL::RotationalInterpolation* rot_interpo = new KDL::RotationalInterpolation_SingleAxis();
   try
   {
-    KDL::Path* circ = new KDL::Path_Circle(start_pose,
+    return std::unique_ptr<KDL::Path>(new KDL::Path_Circle(start_pose,
```
Comment (gavanderhoorn) [inline pilz_trajectory_generation/src/path_circle_generator.cpp:44]: `std::make_unique<..>(..)`?

Comment (agutenkunst) [reply pilz_trajectory_generation/src/path_circle_generator.cpp:44]: c++11 ...

Comment (gavanderhoorn) [reply pilz_trajectory_generation/src/path_circle_generator.cpp:44]: https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique

https://stackoverflow.com/questions/7038357/make-unique-and-perfect-forwarding

Could bring/roll your own.

Comment (martiniil) [reply pilz_trajectory_generation/src/path_circle_generator.cpp:44]: Ah, I didn't know that. Thanks for the hint.

**Review by martiniil — APPROVED** [review]
The value of implementing a surrogate for a missing STL function is debatable. Still merging ...

**Pull request merged**